config/crd/bases/wildfly.org_wildflyservers.yaml

```diff
@@ -63,6 +63,10 @@ spec:
                 minItems: 1
                 type: array
                 x-kubernetes-list-type: set
+              deactivateTransactionRecovery:
```

Comment:
why isn't the same change in the bundle/manifests/wildfly.org_wildflyservers.yaml ?

and why do we even have two files with basically the same content?

Reply:
IIRC, this one is generated automatically with `make manifest`